# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: 66b86f8b6682
# Full Hash: 66b86f8b6682a8eac45ac0ceb2447958acf96201
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 2
# Description:
#   Bug 1667192 - patch 1 - Update to mapped_hyph v0.4.0, to get FFI access to hyphenation table compilation. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91408
# ==============================================================================

diff -r a96912059b4c -r 66b86f8b6682 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Wed Sep 30 12:45:10 2020 +0000
+++ b/toolkit/library/rust/shared/Cargo.toml	Wed Sep 30 12:51:17 2020 +0000
@@ -46,7 +46,7 @@
 neqo_glue = { path = "../../../../netwerk/socket/neqo_glue" }
 rlbox_lucet_sandbox = { version = "0.1.0", optional = true }
 wgpu_bindings = { path = "../../../../gfx/wgpu_bindings", optional = true }
-mapped_hyph = { git = "https://github.com/jfkthame/mapped_hyph.git", tag = "v0.3.0" }
+mapped_hyph = { git = "https://github.com/jfkthame/mapped_hyph.git", tag = "v0.4.0" }
 remote = { path = "../../../../remote", optional = true }
 fog_control = { path = "../../../components/glean", optional = true }
 app_services_logger = { path = "../../../../services/common/app_services_logger" }
